### PR TITLE
ci: infra lint

### LINKED ISSUE #431: Perform linting/scanning on terraform
The terraform in this repo has some terratests running but does not have linting or scanning being performed. We should adopt the common workflow for any tf in this repo - https://github.com/defenseunicorns/uds-common-workflows/blob/main/.github/workflows/terraform-scan.yaml

### PR DESCRIPTION
**Pull request by TristanHoladay**
resolves #431

### CODE REVIEW

File: .github/filters.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ci-dubbd-iac-aws/loki:
+  - ".github/test-infra/ci-dubbd-iac-aws/loki/**"
+ci-dubbd-iac-aws/valero:
+  - ".github/test-infra/ci-dubbd-iac-aws/valero/**"
```
Comment (mjnagel) [inline .github/filters.yml:4]:
```suggestion
ci-dubbd-iac-aws/velero:
  - ".github/test-infra/ci-dubbd-iac-aws/velero/**"
```

File: .github/test-infra/ci-dubbd-iac-aws/loki/main.tf [hunk 2]
```diff
@@ -1,14 +1,22 @@
+# test tf
 provider "aws" {
   region = var.region
 }
 
 terraform {
+  required_version = "1.4.2"
```
Comment (mjnagel) [inline .github/test-infra/ci-dubbd-iac-aws/loki/main.tf:7]: Is there a reason for this specific version (here and in the github install)? I would think we want latest < 1.6.0, which is 1.5.7?

Comment (TristanHoladay) [reply .github/test-infra/ci-dubbd-iac-aws/loki/main.tf:7]: @mjnagel i used 1.4.2 cause someone mentioned they thought was what is used, but 1.5.7 is probably the better option, like you said. I'll update them to 1.5.7.

**Review by MxNxPx — APPROVED** [review]
:ship: :it:

**Pull request merged**